code/script.js

```diff
@@ -1,53 +1,153 @@
-}
+  showMessage("Hello there, what´s your name?", "bot");
+  receiveSound.play();
+};
 
 // Eventlisteners goes here 👇
+const handleNameInput = (event) => {
+  event.preventDefault();
+  // Store the value in a variable so we can access it after we
+  // clear it from the input
+  const name = getInput();
+  values["name"] = name;
+  // showMessage(`Welcome ${name}!`, "bot");
+  setTimeout(showMessage, 1000, `Welcome ${name}!`, "bot");
+  //tar bort så att inte boten frågar igen om namn
+  event.target.removeEventListener("submit", handleNameInput);
+  setTimeout(askAge, 2000);
+};
+//att den frågar om namnet o går vidare till nästa
+function askName() {
+  form.addEventListener("submit", handleNameInput);
+}
+// event.target.removeEventListener("submit", handleNameInput);
+// After 1 second, show the next question by invoking the next function.
+// passing the name into it to have access to the user's name if we want
+// to use it in the next question from the bot.
+// const askNextQuestion = () => {
+// };
+
+const handleAgeInput = (event) => {
+  event.preventDefault();
+  const age = getInput();
+  values["age"] = age;
+  event.target.removeEventListener("submit", handleAgeInput);
+  // Kontrollera om age är en siffra och inte är en tom sträng
+  if (!isNaN(age) && age.trim() !== "") {
+    setTimeout(askGender, 2000);
+    // Visa ett felmeddelande om åldern inte är en siffra
+  } else {
+    showMessage("Please enter a valid age (numeric value).", "bot");
+    setTimeout(askAge, 1000);
+    // När din chattbot är klar med att skriva
+  }
+};
+function askAge() {
+  showMessage(`How old are you?`, "bot");
+  form.addEventListener("submit", handleAgeInput);
+}
+
+const handleGenderInput = (event) => {
+  event.preventDefault();
+  const gender = getInput();
+  values["gender"] = gender;
+  event.target.removeEventListener("submit", handleGenderInput);
+  setTimeout(askHairColor, 2000);
+};
+function askGender() {
+  showMessage(`What's your gender?`, "bot");
+  form.addEventListener("submit", handleGenderInput);
+}
+const hairInput = (event) => {
+  event.preventDefault();
+  const hair = getInput();
+  values["hair"] = hair;
+  event.target.removeEventListener("submit", hairInput);
+  setTimeout(askSport, 2000);
+};
+function askHairColor() {
+  showMessage(`What's your hair color?`, "bot");
+  form.addEventListener("submit", hairInput);
+}
+const sportInput = (event) => {
+  event.preventDefault();
+  const sport = getInput();
+  values["sport"] = sport;
+  event.target.removeEventListener("submit", sportInput);
+  setTimeout(printFinal, 2000);
+};
+function askSport() {
+  showMessage(`Do you practice any sports?`, "bot");
+  form.addEventListener("submit", sportInput);
+}
+
+function printFinal() {
+  showMessage(
+    `OK! This is what I know about you. your name is ${values["name"]}, you are ${values["age"]} years old, your gender is ${values["gender"]}, you have ${values["hair"]} hair and you practice ${values["sport"]}.`,
```

Comment:
Haha, I honestly didn't know how to answer the sport question but I ended up answering "Yes" so your summary didn't work to well on that answer. It would be cool if you added a condition based on that, like - if the answer is yes or no, then the message just says "sport" and otherwise it shows the actual user input (if I would've answered "running" for example)